guides/micronaut-oracle-cloud/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+  "asciidoctor": "micronaut-oracle-cloud.adoc",
+  "slug": "micronaut-oracle-cloud",
+  "title": "Deploy a Micronaut app to Oracle Cloud",
+  "intro": "Learn how to deploy a Micronaut application to Oracle Cloud.",
+  "authors": ["Burt Beckwith"],
+  "tags": ["java", "oracle", "cloud"],
```

Comment:
We don't need the language tags. Please remove `java`.

Reply:
We automatically include language and build tool tags when generating the json with all the metadata: https://github.com/micronaut-projects/micronaut-guides-poc/blob/58c16ddd371255d0ef1ccd8f9d7498cceccbb91a/buildSrc/src/main/groovy/io/micronaut/guides/IndexGenerator.groovy#L249